Makefile.am

```diff
@@ -154,6 +154,8 @@ cpplint_FILES += $(noinst_HEADERS)
 clean-local:
 	-rm -f $(BUILT_SOURCES) $(CLEANFILES)
 
+check: cpplint
+
```

Comment:
Re: `check-local`, I wondered that myself. It does seem more idiomatic, so perhaps that by itself is an argument in its favor.